### PR TITLE
chore(messages): remove unused references to msg metadata

### LINKED ISSUE #5948: 'msg' metadata in messages, what for?
I can't seem to understand what 'msg' metadata represents in the messages plugin and why do we have an extra join for it given that it seems to be attached to all messages...

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #5948

### CODE REVIEW

**Comment by mrclay** [comment]
Please add a brief message in the plugin upgrade docs (the 1.11 to 2.0 section) that messages will no longer have this metadata.

**Comment by jdalsem** [comment]
you mean in a readme? that does not exists yet?

**Comment by juho-jaakkola** [comment]
He means this: http://learn.elgg.org/en/latest/guides/upgrading.html

**Comment by jdalsem** [comment]
oh, but why not in a section from 1.x to 2.0 because there could be a 1.12 (also applies to the current 1.11 to 2.0 section)

**Comment by juho-jaakkola** [comment]
I think the idea of mentioning the exact number is to emphasize the fact that one should upgrade one minor version at a time all the way to the last minor version. (As opposed to going to 2.0.0. from any 1.x.x version.)

Use 1.11 section for now. Let's update that when/if 1.12 gets released.

File: docs/guides/upgrading.rst [hunk 1]
```diff
@@ -3,12 +3,42 @@ Upgrading Plugins
...
 
 .. contents:: Contents
    :local:
    :depth: 2
 
+From 1.11 to 2.0
+================
+
+Breadcrumbs
+-----------
```
Comment (juho-jaakkola) [inline docs/guides/upgrading.rst]: I don't think this part is supposed to be in this PR?

**Comment by jdalsem** [comment]
going to make a new PR... i am totally messing this one up :)

**Comment by jdalsem** [comment]
new PR here #7856

**Pull request closed without merging**